adafruit_atecc/adafruit_atecc.py

```diff
@@ -372,7 +395,8 @@ def _random(self, data):
 
     # SHA-256 Commands
     def sha_start(self):
```

Comment:
we can type the return of `sha_start()` Looks like it will return `status` which is a bytearray.